internal/templates/templatesfunctions.go

```diff
@@ -445,3 +450,25 @@ func formatDuration(seconds int) string {
 
 	return strings.TrimSpace(result)
 }
+
+func makeMap(kvs ...any) map[any]any {
+	m := make(map[any]any)
+	for i := 0; i < len(kvs)-1; i += 2 {
+		m[kvs[i]] = kvs[i+1]
+	}
+
+	return m
+}
+
+func T(msgID string, tplData ...map[any]any) string {
+	lng := language.Make(configs.GetTranslationConfig().Language.String())
+
+	msg, err := i18n.Translate(lng, msgID, tplData...)
+	if err != nil {
+		if !i18n.IsMessageNotFoundErr(err) && !i18n.IsMessageFallbackErr(err) {
+			mudlog.Error(`Translation`, "msgID", msgID, `error`, err)
+		}
+	}
+
+	return msg
+}
```

Comment:
What are your thoughts regarding moving all translation code into a package called:

`language`

The template function would still exist as:

`"t":   language.T,`

We can put all ofthe i18n code into there.

This will make for a more user-friendly package name ( `i18n` is hard to remember ) and allow us to create other useful language/translation related functions and features (such as the `language.Basics().Yes`, `language.Basics().No` etc. mentioned above)